### ISSUE TITLE
strings.json changes no longer showing in build #325

### ISSUE THREAD

**Issue by SecretlyJealous** [IntellectualSites/FastAsyncWorldEdit#584]
# Bug report for FastAsyncWorldEdit 1.16.1

**[REQUIRED] FastAsyncWorldEdit Configuration Files:** 
config.yml: https://pastebin.com/qbx642wn
config-legacy.yml: https://pastebin.com/E4qmuBMb

**[REQUIRED] FastAsyncWorldEdit Version Number:** 
FastAsyncWorldEdit-1.16-325

**[REQUIRED] Spigot/Paper Version Number:** 
git-Paper-131 (MC: 1.16.1) (Implementing API version 1.16.1-R0.1-SNAPSHOT)

**[REQUIRED] Description of the problem:**
When using build 325 the strings.json file doesn't show your own custom changes, I've changed the prefix to "WorldEdit":

```
{
	"prefix": "&8(&4&lWorldEdit&8)&7 {0}",

	"fawe.worldedit.history.find.element": "&2{0} {1} &7ago &3{2}m &6{3} &c/{4}",
	"fawe.worldedit.history.find.element.more": " - Changes: {0}\n - Bounds: {1} -> {2}\n - Extra: {3}\n - Size on Disk: {4}",
	"fawe.worldedit.history.find.hover": "{0} blocks changed, click for more info",
```

However in game it doesn't show any changes I make to this file:
![2020-08-26_13 47 27](https://user-images.githubusercontent.com/10485646/91305261-b2e08c00-e7a2-11ea-8cf3-0e3b2cec593a.png)

**How to replicate**:
Update to build FastAsyncWorldEdit-1.16-325.

**Plugins being used on the server:**
FastAsyncWorldEdit, LuckPerms, PlotSquared, ProtocolLib

**Checklist**:

- [X] I included all information required in the sections above
- [X] I made sure there are no duplicates of this report [(Use Search)](https://github.com/IntellectualSites/FastAsyncWorldEdit/issues?q=is%3Aissue)
- [X] I made sure I am using an up-to-date version of [FastAsyncWorldEdit for 1.16.1](https://ci.athion.net/job/FastAsyncWorldEdit-1.16/)
- [X] I made sure the bug/error is not caused by any other plugin

**Comment by mibby** [comment]
Can also confirm experiencing this with FAWE dev 326. 

More specifically, I remove the (FAWE) prefix and just set ``"prefix": "{0}",`` so (FAWE) doesn't show up when players use the ``/butcher`` command. However it appears forcefully now, ignoring changes set in strings.json.

**Comment by MattBDev** [comment]
@mibby @SecretlyJealous Are the changes ignored even after you restart the server?

**Comment by SecretlyJealous** [comment]
I already had a custom file and when I updated to build 325 in game it was acting as if that file didn't exist, I regenerated it and then tried editing it but it still didn't work. It uses the default stuff.

**Comment by mibby** [comment]
@MattBDev Restarting the server and also reloading with ``/fawe reload`` does not apply anything in the file.

**Comment by aurorasmiles** [comment]
@SecretlyJealous could you test if that issue still exists in the latest build?

**Comment by N0tMyFaultOG** [comment]
Fixed in 4771297add30c15e2c3cfaef7892002034795a5d
